environment.yml

```
@@ -5,8 +5,8 @@ dependencies:
   # Dependencies from setup.cfg
   - python=3.11
   - coloredlogs
-  - jax >= 0.4.13
-  - jaxlib
+  - jax >= 0.4.13,<0.4.15
+  - jaxlib < 0.4.15
```

Shouldn't we put this as in [`setup.cfg`](https://github.com/ami-iit/jaxsim/blob/d0f4d4e0d1e0c0685b37e4f9fcac26c1466fb545/setup.cfg#L56C1-L57C30)? (jax<0.4.25):

```suggestion
  - jax >= 0.4.13,<0.4.25
  - jaxlib < 0.4.25
```